### DISCUSSION TITLE
S866 lcd display wiring diagram for the right way to connect my throttle to my motor controller

### DISCUSSION THREAD

**Discussion by 70Frank666** [community/community#137538] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hélp this is my first project. And please help me fix it. Because I can't find the right way how to connect the wires from my lcd display S866 to my throttle and motor controller

**Comment by LiteBrite82** [comment]
Hi @70Frank666 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
